shared/chat/inbox/container/remote.js

```diff
@@ -90,6 +90,7 @@ export const serialize = ({
     channelname: conversation.channelname,
     conversationIDKey: conversation.conversationIDKey,
     hasBadge,
+    hasBottomLine: true,
```

Comment:
you checked that this worked in teh widget right

Reply:
Yup!